generator.go

```diff
@@ -227,6 +235,12 @@ func (g *DataGenerator) generateInternal(params *GenerateParams) (interface{}, e
 		return listData, err
 	}
 
+	// Generate a synthethic schema as a last ditch effort
+	if example == nil && schema.XResourceID == "" {
+		example = &valueWrapper{value: generateSyntheticFixture(schema, context)}
```

Comment:
Yeah, I like that idea. I ended up adding this code:

``` go

		if verbose {
			// We list properties here because the schema might not have a
			// better name to identify it with.
			fmt.Printf("Generated synthetic fixture with properties: %s\n",
				stringOrEmpty(propertyNames(schema)))
		}
```

I didn't put it in as an error because if we use this for prereleases, we could plausibly expect this to be a pretty standard path. Also, I'm not sure that we can ever expect to have a good fixture for some of our edge resources like "charge outcome rule".